README.md

```diff
@@ -1,2 +1,32 @@
 # CFReT-screen-mAP-analysis
-Analysis notebooks applying mean average Precision (mAP) to the CFReT screen image profile dataset.
+
+## About
+
+Cardiac fibrosis, a condition associated with significant morbidity and mortality, contributes to the global burden of cardiovascular diseases, which remain the leading cause of death worldwide, accounting for approximately 31% of all deaths.
+This pathological process is characterized by the excessive deposition of extracellular matrix (ECM) components, especially collagen, resulting in the formation of scar tissue.
+While ECM production is a vital part of the body’s response to injury, its overproduction disrupts tissue homeostasis and leads to structural changes that impair organ function.
+In the heart, for example, excessive ECM accumulation causes stiffening of the myocardial tissue, compromising its ability to contract and relax efficiently.
+This can lead to reduced cardiac output, impaired electrical conductivity, and an increased risk of heart failure or fatal arrhythmias.
+Given its global impact and contribution to high mortality rates, there is an urgent need to develop advanced diagnostic tools and targeted therapeutic strategies to better understand, manage, and treat cardiac fibrosis.
+
+Recently, we applied image-based profiling with the [CellPainting](https://www.nature.com/articles/nprot.2016.105)(CP) assay to investigate whether single-cell image-based profiles could be leveraged to identify failing heart cells (cardiac fibrosis), healthy heart cells, and heart cells treated with `drug_x`.
+Our study demonstrated that by using image-based profiles into machine learning models, we could accurately distinguish between failing and healthy cells.
+Additionally, we identified morphological features that were critical for the model’s ability to differentiate between cell states.
```

Comment:
```suggestion
Additionally, we identified morphological features that were critical for the model’s ability to differentiate between heart states.
```

Reply:
Recommend not stating that we are looking at cell state since there is a lot of heterogeneity within these cell populations per heart. We are predicting if these cells from heart failure or "healthy" hearts.